Makefile

```diff
@@ -0,0 +1,36 @@
+BUILD = build
+BOOKNAME = monero-gui-wallet-guide
+TITLE = title.txt
+METADATA = metadata.xml
+CHAPTERS = ch00.md ch01.md ch02.md ch03.md ch04.md ch05.md ch06.md ch08.md ch09.md
+TOC = --toc --toc-depth=2
+COVER_IMAGE = media/monero-logo-1280.png
```

Comment:
I see you didn't upload a logo. Is this just a placeholder?

Reply:
Whoops, no I forgot to commit it.

Reply:
> I see you used the documentclass 'book'. I tested it using 'report' and it was looking much better IMO

Which ever template looks best to you is fine with me. My report template must be slightly different than yours...when I tried to use it, I got an error about \frontmatter not being defined.

Reply:
I think `/frontmatter` and ` \mainmatter ` are not recognized by the *report* template. I removed them during my tests, and it worked just fine